src/zcash/zcashTypes.ts

```diff
@@ -64,6 +64,12 @@ export interface ZcashInitializerConfig {
   birthdayHeight: number
 }
 
+export interface ZcashAddresses {
+  unifiedAddress: string
+  saplingAddress: string
+  transparentAddress: string
+}
+
```

Comment:
Can you do `import type {ZcashAddresses } from 'react-native-zcash'`? The `import type` syntax ensures that there are no runtime consequences for the import, only type information.